### PR TITLE
move evochain logic into ownership chain

### PR DESCRIPTION
**Pull request by asiniscalchi**
as a first step our custom logic is moved to the ownership chain

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -14,9 +14,7 @@ members = [
 	"bridge/primitives/*",
    "evolution-chain/runtime",
    "evolution-chain/node",
-   "evolution-chain/pallets/*",
    "evolution-chain/primitives",
```
Comment (magecnion) [inline Cargo.toml:17]: I think we also need to bring primitives from laos-evolution ?

**Review by magecnion — CHANGES_REQUESTED** [review]

File: ownership-chain/precompile/laos-evolution/contracts/LaosEvolution.sol
Comment (magecnion) [inline ownership-chain/precompile/laos-evolution/contracts/LaosEvolution.sol:1]: what about rethinking the names? it's kind of weird that evolution precompiles package is called `laos-evolution` and the ownership `living-assets`

**Review by magecnion — CHANGES_REQUESTED** [review]

File: Cargo.toml (hunk 1 above)
Comment (asiniscalchi) [reply Cargo.toml:17]: there's no dependency from the evolution pallet and precompiles with primitives of evolution chain

File: ownership-chain/precompile/laos-evolution/contracts/LaosEvolution.sol
Comment (asiniscalchi) [reply ownership-chain/precompile/laos-evolution/contracts/LaosEvolution.sol:1]: we discussed to handle this in a future task

**Review by magecnion — APPROVED** [review]

**Pull request merged**